.github/workflows/postman-docker.yaml

```diff
@@ -22,38 +22,56 @@ jobs:
 
       matrix:
         backend: [ cassandra-40, cassandra-311, dse-68 ]
-        collection: [ docsapi-library, graphqlapi-library, restapi-users ]
+        collection: [ graphqlapi-library, restapi-users ]
         include:
           - backend: cassandra-40
             path: docker-compose/cassandra-4.0
           - backend: cassandra-311
             path: docker-compose/cassandra-3.11
           - backend: dse-68
             path: docker-compose/dse-6.8
-          - collection: docsapi-library
-            collection-id: 12949543-caba2a02-6559-486a-9e4a-d5c0791fd296
-            environment-id: 17930693-2f355f7b-123d-4bca-9a9c-7d82222ca49d
+          #- collection: docsapi-library
+          #  collection-id: 12949543-caba2a02-6559-486a-9e4a-d5c0791fd296
+          #  environment-id: 12949543-2e78cf27-bd8c-43f2-909f-70a2b87d65fe
```

Comment:
Temporarily commenting this one out while I work with Lorina on it running successfully to completion (it has extra keyspace creation commands that cause errors).